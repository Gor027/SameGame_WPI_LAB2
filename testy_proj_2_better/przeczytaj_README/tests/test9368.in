((( ( DDD( D  (D((( ( DD((D(D ( ((((( D  
(D (  D(D    D(DDDD (D (((DDDD   ((D((   (D DDD(       (DD ( D D
D(      (DDD DDD(((( DD((D  (DD D(D((D   
DDD ( 
DDD 
D(D( DD(DDD(D(DD((DD  D  (((D (  D  (( D ( DD(DDD((((((DD( D( ((D D (
(DD(   DDDDD(DD(( (DD(DD DDD(D ( D( (((D(D D(D  D (DDD(D(D (  D ( D(
D  (DD ((DD   (D  (( D D D D(((D (DDDDDD
 ( DD DD(D(DD ((  (D DD(( D(( ( (D(DD  D ( D((D  DD  DD ( (D  ( DDD (DD(D((
(DD(  D ( DD D(DDD
  D   ( DD((D  DDD(D ( D (((D DD( (( D D D
(D( D  ((  D
  (DDD D((  DDD(( D(((    (DDD (D D ((DD (DD(D(DD DD(DD (D(D   ( (D
