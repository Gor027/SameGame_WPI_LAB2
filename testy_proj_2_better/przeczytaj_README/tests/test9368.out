        D
 (      D      D
 D(     (     (D  (         D
((D    DD( D  (D( D        (D     ( (
(DD(   (DDDD  DDD (( D D (DD(    (( ((
DDD(( D((DD( DD(((DD D ( (D(D (  (D((D D (
D(D(( DD(DDDDDDD(D(D (D( D(D(DD (((D(( D (  DD          D  (
DDD(D D((DDD(((((((((DDD((DD(D((DDD(((DD D DD(D   (  (( DD D      D
D(DDDDDDDD(DDD(DD((D(DDD(DD((D(D(D(DDD(D ( D(DD(D (((D(((( (  (DD D
(D((DDD((DD(DD(DDD(DD(D((D((D((D(((DDDDD DDD((DDD(DDDDDD(D(D  D((DD (
DD(DDD(D(((DDDD((DD(((DD(((DDDD(DDD(((DD((DD(D(DDDDD(DDD(D(D(D((D(D((DD(D((
